Tests/SwiftLintFrameworkTests/RulesTests.swift

```diff
@@ -2,6 +2,15 @@ import SwiftLintFramework
 import XCTest
 
 class RulesTests: XCTestCase {
+    func testDisableRationale() {
+        verifyRule(
+            DisableRationaleRule.description,
+            commentDoesntViolate: false,
+            skipStringTests: true,
```

Comment:
I disabled the string tests because, even though I used `let disableMatches = file.match(pattern: "swiftlint:disable", with: [.comment])` (note the `.comment`), still violations are raised when executing the string tests. 
I have been looking into it but could not find the reason behind it, could it be a bug inside `let disableMatches = file.match(pattern: "swiftlint:disable", with: [.comment])`?

Reply:
Okey, I think the problem was that my code has double quote characters in it that where not being escaped when converting them to string literal. I think I fixed that, now string tests are also passing.